.github/workflows/test.yml

```
@@ -30,6 +30,10 @@ jobs:
         platform: [ubuntu-latest]
     runs-on: ${{ matrix.platform }}
     steps:
+      - name: Prefer Google mirror globally
+        run: |
+          echo '{"registry-mirrors":["https://mirror.gcr.io"]}' | sudo tee /etc/docker/daemon.json
```

We can rely on ECR and GCR mirrors.

```suggestion
      - name: Set up Docker Hub mirrors
        run: |
          echo '{"registry-mirrors": ["https://mirror.gcr.io", "https://public.ecr.aws"]}' | sudo tee /etc/docker/daemon.json
```